release-notes/unreleased/IGAPP-608-remove-nearby-locations-setting.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: IGAPP-588
+show_in_stores: false
+platforms:
+  - android
+  - ios
+en: The now unused setting proposeNearbyCities has been removed.
```

Comment:
I think this message should be understandable for users and why only in english?
Something with more account to what the user gets/sees like: You are no longer asked to allow access to your location, unless you use/activate specific features.

Reply:
We don't show this message to any users but only include in the release notes posted on mattermost/github as `show_in_stores` is set to false, therefore only English is relevant. I think removing a settings option is not anything that is relevant to users and is more confusing than helping.